packages/migrate/src/__tests__/handlePanic.migrate.test.ts

```diff
@@ -176,13 +168,20 @@ describe('handlePanic migrate', () => {
         draft: false,
         prismaSchema: migrate.getPrismaSchema(),
       })
-    } catch (error) {
+    } catch (e) {
+      const error = e as RustPanic
+
       expect(error).toMatchSnapshot()
       expect(JSON.parse(JSON.stringify(error))).toMatchObject({
         area: 'LIFT_CLI',
         schemaPath,
       })
-      expect(error.rustStack).toContain('This is the debugPanic artificial panic')
+      expect(error.message).toContain('This is the debugPanic artificial panic')
+
+      const isWindows = ['win32'].includes(process.platform)
+      if (!isWindows) {
+        expect(error.rustStack).toContain('std::panicking::')
```

Comment:
Why is windows excluded from the test?

Reply:
Because Rust panics on Windows are something like

```
<unknown>
<unknown>
<unknown>
...
```

Reply:
Does that mean this is going to show up for windows users or are we using the fallback message just for windows users?

Reply:
And is there a no way to have rust stacktraces on windows?

Reply:
Rust stacktraces are known problems on Windows unfortunately. With the "fallbackMessage" (notice that this part of the code changed throughout the night), it would look something like this:

```
[EXIT_PANIC]
${fallbackMessage}
   <unknown>
  <unknown>
  <unknown>
  ...
  <unknown>
```

As specified in the PR description, this is not user-facing, it's just useful for our error reporting system.

Reply:
> As specified in the PR description, this is not user-facing, it's just useful for our error reporting system.
That seems to have been added after I started my review :P Good to know!